test/e2e/specs/fakerp/etcdrecovery.go

```diff
@@ -75,69 +72,12 @@ var _ = Describe("Etcd Recovery E2E tests [EtcdRecovery][Fake][LongRunning]", fu
 		Expect(err).NotTo(HaveOccurred())
 		Expect(cm1.Data).To(HaveKeyWithValue("value", "before-backup"))
 
-		backupImage := "quay.io/openshift-on-azure/etcdbackup:latest"
```

Comment:
I am a bit confused why this chunk was deleted, can you clarify the intent?

Reply:
@thekad this chunk was deleted because all of that stuff will be handled via the admin client. In the RP backend, a job will be created which will perform and validate the backup. In effect, as a user of the admin client, you'll be able to take a backup of an arbitrary cluster's etcd by simply doing:
```
# returns a future which you can poll for completion
admincli.Backup(ctx, resourceGroup, resourceGroup, backupName)
```
or

```
#performs the backup and wait for it to complete before returning
admincli.BackupAndWait(ctx, resourceGroup, resourceGroup, backupName)
```